src/features/transactions/components/txn-list/txn-list-item/multisig-txn-list-item.tsx

```diff
@@ -204,6 +204,8 @@ export function SubmittedMultisigTxnDetails({
   const { memo, executeAutomatically, threshold, transaction, submitter } =
     (multisigTxnInfoData?.info ?? {}) as MultisigTransactionInfo
 
+  const memoStr = memo && typeof memo[0] === "string" ? memo[0] : ""
```

Comment:
I could also do a map here. Depends on how many human-readable memos we expect transactions to have.

Reply:
Just the 1 from the UI for now so I think that's fine.